example_app_generator/generate_action_mailer_specs.rb

```diff
@@ -14,15 +14,10 @@
       end
     end
   CODE
-  gsub_file 'config/initializers/action_mailer.rb',
-            /ExampleApp/,
-            Rails.application.class.parent.to_s
 
   copy_file 'spec/support/default_preview_path'
   chmod 'spec/support/default_preview_path', 0755
-  gsub_file 'spec/support/default_preview_path',
-            /ExampleApp/,
-            Rails.application.class.parent.to_s
+
```

Comment:
I'm convinced these are still needed, its just the Rails 6.1 implementation might need to change?

Reply:
I deep dive a little bit on this few weeks ago. The `Rails.application.class.parent` on previous Rails is `ExampleApp`.  So I didn't understand why the gsub was needed.
Also if the CI pass isn't what we want?